crud/src/test/resources/ITCaseCrudResourceRDBMSTest/config/config.properties

```diff
@@ -1,3 +1,4 @@
 # set default isolation strategy for tests to SEMAPHORE else breakpoints in IDE
 # when debugging unit tests will timeout with default strategy (THREAD).
 hystrix.command.default.execution.isolation.strategy=SEMAPHORE
+execution.isolation.thread.timeoutInMilliseconds=30000
```

Comment:
`execution.isolation.thread.timeoutInMilliseconds` isn't a valid property I think.  Anyway, the default isolation is SEMAPHORE in tests, not THREAD, so this wouldn't have any impact.  See https://github.com/Netflix/Hystrix/wiki/Configuration#execution.isolation.thread.timeoutInMilliseconds